CExcercises/Ex2-offsetOf.c

```diff
@@ -20,7 +20,8 @@ allocables en el sistema)
 #include <stdio.h>
 #include <string.h>
 
-#define offsetOf(STRUCT, FIELD) ((int)(&STRUCT) - (int)(&FIELD))
+//The Field is posicionated in a bigger position of memory, because the 
+#define offsetOf(STRUCT, FIELD) ((int)(&FIELD)-(int)(&STRUCT))
```

Comment:
check the excercise, and answer the last question:
"¿Es posible utilizar esta macro con estructuras que contengan bitfields? ¿Porque?"